scripts/release-build.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+const directory = './build/';
+
+const verbatimFiles = ['LICENSE', 'README.md', 'index.d.ts', 'types.d.ts'];
+
+// Returns a string representing data ready for writing to JSON file
+function createDataBundle() {
+  const bcd = require('../index.js');
+  const string = JSON.stringify(bcd);
+  return string;
+}
+
+// Returns a promise for writing the data to JSON file
+function writeData() {
+  const dest = path.resolve('./', directory, './data.json');
+  const data = createDataBundle();
+  const promise = fs.writeFile(dest, data);
+  return promise;
+}
+
+// Returns an array of promises for copying of all files that don't need transformation
+function copyFiles() {
+  let promises = [];
+  for (const file of verbatimFiles) {
+    const src = path.join('./', file);
+    const dest = path.join(directory, file);
+    const promise = fs.copyFile(src, dest);
+    promises = promises.concat(promise);
+  }
+  return promises;
+}
+
+function createManifest() {
+  const full = require('../package.json');
+  const minimal = {};
```

Comment:
```suggestion
  const minimal = {main: 'data.json'};
```

I couldn't load the built module without this.

Reply:
Done!